conbench/tests/entities/test_history.py

```diff
@@ -347,6 +348,52 @@ def test_set_z_scores_with_distribution_change(
         assert_equal_leeway(benchmark_result.z_score, expected_z_score)
 
 
+@pytest.mark.parametrize("data", [[[1], [2], [5]], [[10, 20], [11, 22], [50, 70]]])
+@pytest.mark.parametrize("unit", ["s", "i/s"])
+@pytest.mark.parametrize("svs_type", ["mean", "best"])
+def test_set_z_scores_with_small_reps(data: List[List[int]], unit: str, svs_type: str):
+    expected_z_scores = {
+        (1, "s", "mean"): [None, None, -4.95],
+        (1, "s", "best"): [None, None, -4.95],
+        (1, "i/s", "mean"): [None, None, 4.95],
+        (1, "i/s", "best"): [None, None, 4.95],
+        (2, "s", "mean"): [None, None, -41.72],
+        (2, "s", "best"): [None, None, -55.86],
+        (2, "i/s", "mean"): [None, None, 41.72],
+        (2, "i/s", "best"): [None, None, 34.65],
```

Comment:
I did confirm that the tests where reps=2 and svs_type=best fail before this PR and pass now.